http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/model/LeonardoModel.scala

```diff
@@ -292,55 +209,51 @@ case class DefaultLabels(clusterName: ClusterName,
 
 // Provides ways of combining MachineConfigs with Leo defaults
 object MachineConfigOps {
-  case class NegativeIntegerArgumentInClusterRequestException()
+  case object NegativeIntegerArgumentInClusterRequestException
       extends LeoException(
         s"Your cluster request should not have negative integer values. Please revise your request and submit again.",
         StatusCodes.BadRequest
       )
 
-  case class OneWorkerSpecifiedInClusterRequestException()
-      extends LeoException(
-        "Google Dataproc does not support clusters with 1 non-preemptible worker. Must be 0, 2 or more."
-      )
-
   implicit private val machineConfigSemigroup = new Semigroup[MachineConfig] {
```

Comment:
this is the main change that needs review.

Those negative checks are now in clusterRequestDecoder.